src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2975,7 +2975,7 @@ static void _svgLoaderParserXmlOpen(SvgLoaderData* loader, const char* content,
     if (!attrs) {
         //Parse the empty tag
         attrs = content;
-        while ((attrs != nullptr) && *attrs != '>') attrs++;
+        while ((attrs != nullptr) && strncmp(attrs, "/>", 2) != 0) attrs++;
```

Comment:
if attrs ends at "/", it would occur crash.

Reply:
@hermet please check now - I tried to crash it, but no success :)
by definition, if ```empty == true```  the two last signs are "\>". This is ensured by the xml parser - for so called empty elements type ```SimpleXMLType::OpenEmpty``` is set, and for this type empty is set to be true. that's why it is safe to do ```attrs--```.

Reply:
@mgrudzinska ok thanks.